Silicon/ApollolakePkg/Include/CpuRegs.h

```diff
@@ -244,6 +244,9 @@
 #define B_IA32_MTRR_CAP_SMRR_SUPPORT                                  BIT11
 #define B_IA32_MTRR_CAP_EMRR_SUPPORT                                  BIT12
 
+#define MSR_IA_UNTRUST                                                0x00000120
```

Comment:
#define EFI_MSR_POWER_MISC               0x00000120
#define B_EFI_MSR_POWER_MISC_ENABLE_IA_UNTRUSTED_MODE              (1 << 6)

Reply:
OK. will update, but will remove EFI to align with other MSR register definition.

Reply:
Just aware of these are defined already in this file. so will not update this file. and remove the code where consumes this MSR in EndOfFimware().